<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upload</class>
 <widget class="QDialog" name="upload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="upload_car_name">
      <property name="placeholderText">
       <string>Car</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="upload_track_name">
      <property name="placeholderText">
       <string>Track</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="upload_comment">
      <property name="placeholderText">
       <string>Comment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="upload_link">
      <property name="placeholderText">
       <string>Link</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
